.github/workflows/build_and_test.yml

```diff
@@ -142,22 +192,31 @@ jobs:
             EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DENFORCE_VSX=ON -DENFORCE_VSX3=ON"
           elif [[ ${{ matrix.arch }} = "s390x" ]]; then
             EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DENFORCE_VXE=ON"
-            # Disable VXE2 support, QEMU doesn't support it
+            # Disable VXE2 support, QEMU doesn't support some instructions generated by gcc or llvm
             EXTRA_CMAKE_FLAGS="${EXTRA_CMAKE_FLAGS} -DDISABLE_VXE2=ON"
```

Comment:
Comment and option don't seem to match. What is intent here? Was the problem resolved?